src/platform/np_spake2.h

```diff
@@ -24,7 +24,6 @@ struct nc_spake2_password_request {
 
 
 struct np_spake2_module {
-    np_error_code (*create)(struct np_platform* pl, struct np_spake2_context** spake);
     void (*destroy)(struct np_spake2_context* spake);
```

Comment:
if destroy is not used we should remove it, and we should remove the np_spake2_context as it is not used. If we discover a problem in the future where we need the context then we will add it.